ruby/vampires.rb

```diff
@@ -0,0 +1,101 @@
+#Kane Lee
+
+puts "How many employees are being processed today?"
+employee = gets.chomp.to_i
+
+
+i = 0
+while i < employee
+	
+	puts "What is your name?"
+	name = gets.chomp.downcase
+
+	puts "How old are you? What year were you born?"
+	puts "Age?"
+	age = gets.chomp.to_i
+	puts "Year?"
+	year = gets.chomp.to_i
+
+	# puts "How old are you?"
+	# age = gets.chomp.to_i
+
+	# puts "What year were you born?"
+	# dob = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? yes/no"
+	order = gets.chomp.downcase
+
+	puts "Would you like to enroll in the company's health insurance? yes/no"
+	insurance = gets.chomp.downcase
+
+
+	puts "Can you list your allergies one by one? If you have none then type done."
+
+	allergy = ""
+	while allergy != "sunshine"
+		allergy = gets.chomp.downcase
+		if allergy == "sunshine"
+			break
+		elsif allergy == "done"
+			break
+		else
+			puts "Another one?"
+		end
+	end
+
+
+	vampire_meter = 0
+	if name == "drake cula"|| name == "tu fang"

```

Comment:
Right now, you are testing the different cases, assigning an arbitrary value to the vampire meter and the running a case statement based on that meter. You could instead update the vampire_meter variable with the string you want to return as you run through the conditions.